### DISCUSSION TITLE
클라이언트 서버 관련 질문

### DISCUSSION THREAD

**Discussion by fkdl0048** [fkdl0048/Mentoring-Project#34] [Q&A] [answered]
현재 게임 클라이언트 개발자를 희망하고 있어서 따로 서버를 공부하지 않으려고 했는데 기본적인 개념은 알고 있어야 할 것 같다는 생각이 들었습니다..!  

클라이언트 개발자가 서버를 알아야 한다면 어느정도 개념일까요?

1. 범용적으로 적용되는 서버의 개념
2. 게임클라이언트 서버의 연결 개념
3. 서버를 한번이라도 구현해본다..?

**Comment by jongfeel** [comment, marked as answer]
저라면 1, 2, 3을 다 하는 게 좋다고 봅니다.

아예 모르는 건 안되고 적어도 1 정도는 알고 있어야 합니다.
REST API나 web socket을 어떻게 사용하는지 정도는 되야 클라이언트 구현을 할 수 있으니까요.

**Reply by fkdl0048** [reply]
아직은 클라이언트도 많이많이 부족해서 클라이언트쪽이 익숙해지면 꼭 공부해보겠습니다..!  

~~그때는 서버 프로젝트로 멘토링을..~~
